src/main/java/org/testng/TestNG.java

```diff
@@ -910,6 +916,7 @@ private void initializeConfiguration() {
       //
       for (XmlMethodSelector methodSelector : s.getMethodSelectors() ) {
         addMethodSelector(methodSelector.getClassName(), methodSelector.getPriority());
```

Comment:
I don't catch why we need the selector in both m_methodDescriptors and m_selectors. I think we will have selector duplication after https://github.com/cbeust/testng/pull/1675/files#diff-c89ea27c996c668601a33f887b5675f2R1270
Why do we want it?

Reply:
@juherr - They both are different from what I have seen in the codebase. `m_methodDescriptors` basically captures only those method selectors for which a classname and priority has been defined. So I believe these are limited to `org.testng.IMethodSelector` implementations that a end-user can wire in. `m_selector` is basically for beanshells, which TestNG injects from within, whenever it sees a beanshell expression.